ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -280,6 +282,16 @@ UADeepLinkDelegate, UAPushNotificationDelegate, UAInboxDelegate {
             result(nil)
         }
     }
+    
+    private func loadCustomNotificationCategories() {
+        guard let categoriesPath = Bundle.main.path(forResource: "UACustomNotificationCategories", ofType: "plist") else { return }
+        let customNotificationCategories = UANotificationCategories.createCategories(fromFile: categoriesPath) as! Set<UANotificationCategory>
+    
+        if customNotificationCategories.count != 0 {
```

Comment:
I personally prefer "> 0" with counts. Just a suggestion. Feel free to leave as is.
```suggestion
        if customNotificationCategories.count > 0 {
```

Reply:
No, it should be `!= 0`